src/script_compiler.rs

```diff
@@ -61,7 +62,17 @@ extern "C" {
 /// Source code which can then be compiled to a UnboundScript or Script.
 #[repr(C)]
 #[derive(Debug)]
-pub struct Source([usize; 8]);
+pub struct Source {
+    _source_string:          usize,
```

Comment:
The indentations don't look quite right to. You could fix these by running `cargo fmt`.

Reply:
Thanks! I've fixed this. And thank you for kicking off the workflow run @littledivy.